Notes-CaaSP.adoc

```diff
@@ -113,203 +114,222 @@ parameters:
+NOTE: The default is to provide 40 GB of disk space to the CaaS Platform nodes. This is not sufficient for CAP. So use a machine flavor with a bigger disk and let it use the additional free space for storing images (Commands on how to resize the CaaS Platform node root fs are provided in the instructions below).
 
 
-### Initial preparations
+==== Initial preparations
 
 The following steps only have to be done once before the initial CaaS Platform deployment. For following deployments of CaaS Platform this has no to be redone but already created elements can be reused. The deployment of CaaS Platform on OpenStack is done using existing Terraform rules with some additional steps required for running CAP on the CaaS Platform deployment.
 
 
 * Start with downloading and sourcing the openrc.sh file for OpenStack API access
+[source,bash]
+----
+$ firefox https://$OPENSTACK/project/access_and_security/api_access/openrc/
+. openrc.sh
+----
 
-  ```
-  firefox https://$OPENSTACK/project/access_and_security/api_access/openrc/
-  . openrc.sh
-  ```
+NOTE: You will have to login to download the file. The filename might have a prefix named after the OpenStack project the file is for.
 
-  *Note* You will have to login to download the file. The filename might have a prefix named after the OpenStack project the file is for.
 
-
-**Optional steps**
+===== Optional steps
 
 These step can be performed but are not mandatory. You can also use already existing OpenStack objects instead (e.g. if you do not have the permission to create projects or networks).
 
 * Create an OpenStack project to run CaaS Platform in (e.g. caasp), add a user as admin and export the project to be used by Terraform
 
-  ```
-  openstack project create --domain default --description "CaaS Platform Project" caasp
-  openstack role add --project caasp --user admin admin
-  export OS_PROJECT_NAME='caasp'
-  ```
+[source,bash]
+----
+$ openstack project create --domain default --description "CaaS Platform Project" caasp
+$ openstack role add --project caasp --user admin admin
+$ export OS_PROJECT_NAME='caasp'
+----
 
 * Create a OpenStack network plus a subnet for caasp (e.g. caasp-net) and add a router to the external (e.g. floating) network
 
-  ```
-  openstack network create caasp-net
-  openstack subnet create caasp_subnet --network caasp-net --subnet-range 10.0.2.0/24
-  openstack router create caasp-net-router
-  openstack router set caasp-net-router --external-gateway floating
-  openstack router add subnet caasp-net-router caasp_subnet
-  ```
+[source,bash]
+----
+$ openstack network create caasp-net
+$ openstack subnet create caasp_subnet --network caasp-net --subnet-range 10.0.2.0/24
+$ openstack router create caasp-net-router
+$ openstack router set caasp-net-router --external-gateway floating
+$ openstack router add subnet caasp-net-router caasp_subnet
+----
 
```

Comment:
Here again, for the following paragraphs/bullets/indent, you can keep the indent with a plus:

```
* Upload the CaaS Platform v2 image to OpenStack
+
You can get the SUSE-CaaS-Platform-2.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2 image link:https://download.suse.com/Download?buildid=tW8sXCIHrWE~[here] (SUSE customer account required).
+
[source,bash]
----
$ openstack image create --file SUSE-CaaS-Platform-2.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2 SUSE-CaaS-Platform-2.0-GM
----

* Create a additional security group with rules needed for CAP
+
[source,bash]
----
$ openstack security group create cap --description "Allow CAP traffic"
$ openstack security group rule create cap --protocol any --dst-port any --ethertype IPv4 --egress
$ openstack security group rule create cap --protocol any --dst-port any --ethertype IPv6 --egress
$ openstack security group rule create cap --protocol tcp --dst-port 20000:20008 --remote-ip 0.0.0.0/0
$ openstack security group rule create cap --protocol tcp --dst-port 443:443 --remote-ip 0.0.0.0/0
$ openstack security group rule create cap --protocol tcp --dst-port 2793:2793 --remote-ip 0.0.0.0/0
$ openstack security group rule create cap --protocol tcp --dst-port 4443:4443 --remote-ip 0.0.0.0/0
$ openstack security group rule create cap --protocol tcp --dst-port 80:80 --remote-ip 0.0.0.0/0
$ openstack security group rule create cap --protocol tcp --dst-port 2222:2222 --remote-ip 0.0.0.0/0
----
```

aso.